schematic/models/metadata.py

```diff
@@ -187,7 +202,7 @@ def get_component_requirements(
 
     # TODO: abstract validation in its own module
     def validateModelManifest(
-        self, manifestPath: str, rootNode: str, restrict_rules: bool = False, jsonSchema: str = None, 
+        self, manifestPath: str, rootNode: str, restrict_rules: bool = False, jsonSchema: str = None, project_scope: str = None,
```

Comment:
Let's have project_scope as a list here (for consistency with other calling functions). Perhaps parsing the CLI project_scope arg into a list can be done in the commands.py